pkg/apis/application/v1alpha1/types.go

```diff
@@ -143,6 +143,8 @@ type ApplicationSource struct {
 	Plugin *ApplicationSourcePlugin `json:"plugin,omitempty" protobuf:"bytes,11,opt,name=plugin"`
 	// Chart is a Helm chart name
 	Chart string `json:"chart,omitempty" protobuf:"bytes,12,opt,name=chart"`
+	// RepoNamespace is the namespace of Helm OCI-based repositoty
+	RepoNamespace string `json:"repoNamespace,omitempty" protobuf:"bytes,13,opt,name=repoNamespace"`
```

Comment:
Is it possible to keep using chart name instead of adding `RepoNamespace` field? So that user might provide specify `myrepository/my-chart` as a chart name.

There is a [GetHelmCharts](https://github.com/argoproj/argo-cd/blob/master/server/repository/repository.go#L219) API that is used to suggest available helm charts in the specified Helm repository. If we re-use `Chart` to identify chart in the repository then we don't need to introduce   `RepoNamespace` field in `GetHelmCharts` API output and update UI that uses it.